hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -66,14 +66,17 @@
       milestone: V2
       tests: ["flash_ctrl_hw_rma"]
     }
-    {
+    {    
```

Comment:
It seems there are trailing spaces here.

Reply:
Fixed